Xamarin.Forms.Core/Internals/ResourceLoader.cs

```diff
@@ -53,6 +53,17 @@ public class ResourceLoadingResponse
 			public bool UseDesignProperties { get; set; }
 		}
 
-		internal static Action<Exception> ExceptionHandler { get; set; }
+		static Action<Exception> exceptionHandler1;
+
+		[Obsolete("2 is better than 1")]
+		internal static Action<Exception> ExceptionHandler {
```

Comment:
An obsolete internal method? Isn't that funny? Is something stopping us from updating all the call sites to use the new and improve `ExceptionHandler`?

Reply:
some people (you know who you are, I'm looking at you 👀) are using our internals methods with different versions of Forms. So removing this would make that (👀 that one) tool fail with newer version of forms, as we don't release at the same cycle